aerogear/unifiedpush-operator
# feat(CI/CD + Coveralls)

## Motivation 

## What
- Replace Trevis by Circle Ci
- Add automatic tag release process
- Add tag release and image process steps in the README
- Add health badges and coveralls

## Why
- Following the standard (https://github.com/aerogear/standards)
- Create the images automatically which is a requirement for the projects
- Good practices. 
- Add the first unit test files

## Verification Steps
- Check that it worked with success here: https://circleci.com/gh/camilamacedo86/unifiedpush-operator/6
- Check that the coveralls were added with success as well. 

## Checklist:


## Progress


## Additional Notes
<img width="1161" alt="Screenshot 2019-07-02 at 16 25 03" src="https://user-images.githubusercontent.com/7708031/60540675-1d548400-9ce6-11e9-9e4b-2551724ea4ce.png">
